README.md

```diff
@@ -14,7 +14,9 @@ module "aws-sg" {
   egress_cidr_blocks  = [
     "0.0.0.0/0"
   ]
-  inress_ports = [8080, 9090]
+  ingress_ports = [8080, 9090]
+  ingress_protocol = "tcp"
+  engress_protocol = "all"
```

Comment:
Just a tiny thing but shouldnt this be "egress_protocol"? I think you have an extra "n" there

Reply:
Good catch! I'm really not a fan of these examples in the READMEs because they're hard to write and maintain. But they are useful so 🤷